react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react';
+
+const ToggleSwitch = () => {
+  const [dark, setDark] = useState(false);
+  const handleClick = () => {
+    setDark(!dark);
+  };
+  function getButtonMode() {
+    if (dark === false) return 'grey';
+    if (dark === true) return 'green';
+  }
+  return (
+    <div className={`button-house button-${getButtonMode()}`} onClick={handleClick}>
+      <button type="button" role="switch" onClick={handleClick} id="button" className={`button ball-${getButtonMode()}`} />
```

Comment:
On line 15 you use an inline ternary, but here you use a function call. Wouldn't an inline ternary here work as well as below, and be more succinct?